lib/acts_as_list/active_record/acts/sequential_updates_method_definer.rb

```diff
@@ -0,0 +1,23 @@
+module ActiveRecord::Acts::List::SequentialUpdatesDefiner #:nodoc:
+  def self.call(caller_class, column, sequential_updates_option)
+    caller_class.class_eval do
+      @sequential_updates = nil
+
+      define_method :sequential_updates? do
```

Comment:
`class_eval` allows us to just write normal Ruby to be executed within the scope of the class. So you can just do

```ruby
caller_class.class_eval do
  
  private
  
  def sequential_updates?
    ...
  end
end
```

`define_method` is primarily used to define a method with a string interpolated variable name as far as I've seen it. Give it a go and let me know if it doesn't work as expected.